src/components/cart/cart-item/cart-item.tsx

```diff
@@ -15,67 +10,30 @@ export interface CartItemProps {
 }
 
 export const CartItem = ({ cartItem, className, isLast }: CartItemProps) => {
-    const name = cartItem.productName?.translated || '';
-    const imageUrl = getImageHttpUrl(cartItem.image, 120, 120);
-
     const { trigger: updateQuantity } = useUpdateCartItemQuantity();
     const { trigger: removeItem } = useRemoveItemFromCart();
 
-    function updateQuantityHandler(e: ChangeEvent<HTMLInputElement>) {
+    const handleQuantityChange = (e: ChangeEvent<HTMLInputElement>) => {
         if (!cartItem._id) {
             return;
         }
         const newQuantity = parseInt(e.target.value, 10);
         if (newQuantity > 0) {
             updateQuantity({ id: cartItem._id, quantity: newQuantity });
         }
-    }
+    };
 
-    const isAvailable = isCartItemAvailable(cartItem);
+    const handleRemoveItem = () => {
```

Comment:
can be 1 liner

const handleRemoveItem = () =>removeItem(cartItem._id!)

Reply:
hate oneliners

Reply:
you just don't know how to cook them

Reply:
![images](https://github.com/user-attachments/assets/503f5e2b-98fc-431b-93c1-4e99a9361724)